### DISCUSSION TITLE
code highlighting with the tufte-template?

### DISCUSSION THREAD

**Discussion by MichelDSchmid** [quarto-dev/quarto-cli#4340] [Q&A]
Hi all

Is it possible to use code highlighting with the tufte-template? I mean this project template from here
https://fredguth.github.io/tufte-quarto/

I can see the code highlighting in the HTML output, but not in the pdf output.
I think the problem will be in the latex-tufte template which is used from the quarto-tufte template.

So I will check the pandoc conversion first and than the latex-tufte template, some other tips?

many thanks in advance

Mike

**Comment by cderv** [comment]
> Is it possible to use code highlighting with the tufte-template?

What is the tufte-template ? 

You'll probably have more help if you can share more information of what you are trying and what is not working exactly. 

THanks

**Comment by MichelDSchmid** [comment]
So far I could figure out that it seems to be a problem of the extension "include-code-files". 
When I put some code without this extension, this will also appear in the pdf output, but not if I load it via "include-code-files".

I can't see the content of the code file in the latex source .... it will be removed before producing latex output
BUT to remember, in the HTML output, everything seems to be okay...

Any tips where to search?

Thanks in advance
Mike

**Reply by cderv** [reply]
Please fill an issue with example that I can try on my side in https://github.com/quarto-ext/include-code-files 

thank you
